tests/unit_tests/dagmc/test_model.py

```diff
@@ -254,3 +257,121 @@ def test_dagmc_xml(model):
 
     for xml_mats, model_mats in zip(xml_dagmc_univ._material_overrides.values(), dag_univ._material_overrides.values()):
         assert all([xml_mat.id == orig_mat.id for xml_mat, orig_mat in zip(xml_mats, model_mats)])
+
+def test_dagmc_vacuum(request, run_in_tmpdir, tmpdir):
+    # Required initial mats.
+    mats = {}
+    mats["41"] = openmc.Material(name="41")
+    mats["41"].add_nuclide("U235", 0.03)
+    mats["41"].add_nuclide("U238", 0.97)
+    mats["41"].add_nuclide("O16", 2.0)
+    mats["41"].set_density("g/cm3", 10.0)
```

Comment:
```suggestion
    mats["41"].add_nuclide("U235", 1)
    mats["41"].set_density("g/cm3", 0.03)
```
simpler material with same U235 atoms per cm3